Tests/MapboxNavigationTests/NavigationGeocodedPlacemarkTests.swift

```diff
@@ -0,0 +1,85 @@
+import XCTest
+import CarPlay
+import MapboxNavigation
+@testable import TestHelper
```

Comment:
`TestHelper` is supposed to be used in tests only so if any API is  internal but needed in tests we can just make this API public.

Reply:
Yep, removed.